doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -55,7 +55,7 @@ Simulation and IO Control
       set time end           = 1.3    # End time of simulation
       set output name        = out    # Prefix for VTU outputs
       set output frequency   = 1      # Frequency of simulation output
-      set subdivision        = 1      # Mesh subdivision when outputend
+      set subdivision        = 1      # Mesh subdivision when output end
```

Comment:
Subdivision=1 does not do anything, it does not subdivide at all. Consequently, better remove this line man...